### PR TITLE
Update LocalizationManager.cs

### PR DESCRIPTION
**Pull request by WWWesten**
Synchronize language names list with jelliyfin-web and language codes in list with language resource file names. Native names of the languages, <s>exposed the language codes,</s> order by the language codes.

**Changes**

**Issues**

### CODE REVIEW

**Comment by WWWesten** [comment]
Related to https://github.com/jellyfin/jellyfin-web/pull/3171

**Review by crobibero — APPROVED** [review]

**Comment by WWWesten** [comment]
@cvium Can You take a moment and review the code?

**Review by cvium — APPROVED** [review]

**Comment by WWWesten** [comment]
Hi @joshuaboniface, can You take a look and merge PR? Thanks.

File: Emby.Server.Implementations/Localization/LocalizationManager.cs [hunk 1]
```diff
@@ -372,61 +372,76 @@ private static string GetResourceFilename(string culture)
...
-            yield return new LocalizationOption("Bulgarian (Bulgaria)", "bg-BG");
-            yield return new LocalizationOption("Catalan", "ca");
-            yield return new LocalizationOption("Chinese (Hong Kong)", "zh-HK");
-            yield return new LocalizationOption("Chinese Simplified", "zh-CN");
-            yield return new LocalizationOption("Chinese Traditional", "zh-TW");
-            yield return new LocalizationOption("Croatian", "hr");
-            yield return new LocalizationOption("Czech", "cs");
-            yield return new LocalizationOption("Danish", "da");
-            yield return new LocalizationOption("Dutch", "nl");
+            yield return new LocalizationOption("Afrikaans (af)", "af");
```
Comment (Bond-009) [inline Emby.Server.Implementations/Localization/LocalizationManager.cs:375]: Why the ` (af)` here?

**Review by Bond-009 — CHANGES_REQUESTED** [review]

File: Emby.Server.Implementations/Localization/LocalizationManager.cs (hunk 1 above)
Comment (WWWesten) [reply Emby.Server.Implementations/Localization/LocalizationManager.cs]: Ooops!

**Review by Bond-009 — APPROVED** [review]

**Pull request merged**